apps/src/music/blockly/MusicBlocklyWorkspace.ts

```diff
@@ -103,6 +104,7 @@ export default class MusicBlocklyWorkspace {
       readOnly: isReadOnlyWorkspace,
       useBlocklyDynamicCategories: true,
       rtl: isRtl,
+      editBlocks: getAppOptionsEditBlocks(),
```

Comment:
TIL we can just pass through `editBlocks` to our wrapper - seeing that it sets `startMode` to true which is checked in a bunch of places internally. very cool!